<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="5" >
    <widget class="QTabWidget" name="tabs" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_6" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>467</width>
        <height>185</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>EPUB</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="0" >
        <spacer name="verticalSpacer_25" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>241</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="groupBox_20" >
         <property name="title" >
          <string>Backup files</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" >
          <item row="0" column="0" >
           <widget class="QCheckBox" name="backup_c" >
            <property name="text" >
             <string>Create backup files</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <layout class="QFormLayout" name="formLayout" >
            <item row="0" column="0" >
             <widget class="QLabel" name="backup_l" >
              <property name="text" >
               <string>Backup file extension</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QLineEdit" name="backup_e" />
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" >
        <spacer name="verticalSpacer_26" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>240</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="2" >
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>58</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" >
    <spacer name="horizontalSpacer_4" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>101</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="cancel_b" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <spacer name="horizontalSpacer_5" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>101</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3" >
    <widget class="QPushButton" name="ok_b" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" >
    <spacer name="horizontalSpacer_6" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>101</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="5" >
    <widget class="QLabel" name="errstr" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
